documentation/ingles/aodvv2_process.md

```diff
@@ -0,0 +1,128 @@
+<p align="center">
+  <a href="https://locha.io/">
+  <img height="200px" src="pictures/LogotipoTurpial-Color.20-09-19.svg">
```

Comment:
```suggestion
  <img height="200px" src="../../images/LogotipoTurpial-Color.20-09-19.svg">
```

With this you can remove `pictures/LogotipoTurpial-Color.20-09-19.svg`

Reply:
It's not tested, but's worth a try, so you don't have to duplicate the .svg files.

Reply:
it was fixed